strider/fetcher.py

```diff
@@ -116,11 +116,17 @@ async def setup(
             {"query_graph": qgraph},
             self.preferred_prefixes,
         ))["query_graph"]
+
         # Fill in missing predicates with most general term
         for eid, edge in self.qgraph['edges'].items():
             if ('predicate' not in edge) or (edge['predicate'] is None):
                 edge['predicate'] = 'biolink:related_to'
 
+        # Fill in missing categories with most general term
+        for node in self.qgraph['nodes'].values():
+            if ('category' not in node) or (node['category'] is None):
```

Comment:
This might be worth a separate issue, but we should be consistent with how we check for existence in a dictionary throughout the code. My preference is

```python
if not node.get('category', None):
```

because it covers both cases with a single condition. 

Alternatively, we could delete all None valued keys when we receive a query graph and then check for existence with the standard 
```python
'category' in node
```

I think this would be the best solution overall to keep the number of issues down to a minimum.

Reply:
I agree, it might be best if this None == missing assumption only had to happen one place.